src/utilities/src/string_utils.C

```diff
@@ -49,5 +49,27 @@ namespace GRINS
           pos = input.find(delimiter, first_pos);
         }
     }
+
+    void split_string_real( const std::string& input,
+                            const std::string& delimiter,
+                            std::vector<libMesh::Real>& results )
+    {
+      // Skip delimiters at beginning.
```

Comment:
But then I need to create a temporary vector of type `string` to pass to `split_string()`, and then convert and copy all the values from that to `results`. Seems inefficient to me. Or did you have a different idea?